java/clients/src/test/resources/reports/compaction/job/standard/range/mixedJobs.txt

```diff
@@ -3,13 +3,13 @@ Compaction Job Status Report
-| PENDING     | 2022-09-17T13:33:12.001Z | job11111-1111-1111-1111-111111111111 |           1 | partnAAA-AAAA-AAAA-AAAA-AAAAAAAAAAAA |                                      |                          |                          |          |              |                 |               |                |
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| STATE       | CREATE_TIME              | JOB_ID                               | INPUT_FILES | PARTITION_ID                         | TASK_ID                              | START_TIME               | FINISH_TIME              | COMMIT_TIME              | DURATION | RECORDS_READ | RECORDS_WRITTEN | READ_RATE (s) | WRITE_RATE (s) |
+| FINISHED    | 2022-09-23T13:33:12.001Z | job77777-7777-7777-7777-777777777777 |           1 | partnGGG-GGGG-GGGG-GGGG-GGGGGGGGGGGG | task1111-1111-1111-1111-111111111111 | 2022-09-23T13:34:12.001Z |                          | 2022-09-23T13:36:12.001Z |          |              |                 |               |                |
+| FINISHED    | 2022-09-22T13:33:12.001Z | job66666-6666-6666-6666-666666666666 |           1 | partnFFF-FFFF-FFFF-FFFF-FFFFFFFFFFFF | task1111-1111-1111-1111-111111111111 | 2022-09-22T13:34:12.001Z | 2022-09-22T13:35:12.001Z | 2022-09-22T13:36:12.001Z |       1m |          600 |             300 |         10.00 |           5.00 |
+| UNCOMMITTED | 2022-09-21T13:33:12.001Z | job55555-5555-5555-5555-555555555555 |           1 | partnEEE-EEEE-EEEE-EEEE-EEEEEEEEEEEE | task1111-1111-1111-1111-111111111111 | 2022-09-21T13:34:12.001Z | 2022-09-21T13:35:12.001Z |                          |       1m |          600 |             300 |         10.00 |           5.00 |
+| FAILED      | 2022-09-20T13:33:12.001Z | job44444-4444-4444-4444-444444444444 |           1 | partnDDD-DDDD-DDDD-DDDD-DDDDDDDDDDDD | task1111-1111-1111-1111-111111111111 | 2022-09-20T13:34:12.001Z | 2022-09-20T13:35:12.001Z |                          |       1m |            0 |               0 |          0.00 |           0.00 |
+| FINISHED    | 2022-09-19T13:33:12.001Z | job33333-3333-3333-3333-333333333333 |           1 | partnCCC-CCCC-CCCC-CCCC-CCCCCCCCCCCC | task1111-1111-1111-1111-111111111111 | 2022-09-19T13:34:12.001Z | 2022-09-19T13:35:12.001Z |                          |       1m |          600 |             300 |         10.00 |           5.00 |
```

Comment:
We probably want to update the test data and reporting tests to reflect the fact that compaction jobs are always committed by a separate update. It's quite confusing now to test against a compaction job run that's fully finished without a commit update, when that shouldn't be able to happen any more.

Reply:
It looks like a lot of the tests have been adjusted but they're setting the committedBySeparateUpdate incorrectly. Note that the compactionFinishedStatus helper method sets committedBySeparateUpdate to true.